_posts/2020-01-01-2019-summary.md

```diff
@@ -0,0 +1,230 @@
+[Presto Software Foundation]({% post_url 2019-01-31-presto-software-foundation-launch %}), 
+with the long term goal of ensuring the project remains collaborative, open and independent from 
+any corporate interest, for years to come.
+
+Since then, the community around Presto has grown and consolidated. We’ve seen contributions 
+from more than 120 people across over 20 companies. Every week, 280 users and developers 
+interact in the project’s [Slack channel](/slack.html). We'd like to take the opportunity to thank 
+everyone that contributed the project in one way or another. Presto wouldn't be what it is without your 
+help.
+
+With the collaboration of companies such as [Starburst](https://starburstdata.com), [Qubole](https://qubole.com), 
+[Varada](https://varada.io), [Twitter](https://twitter.com), [ARM Treasure Data](https://www.treasuredata.com),
+[Wix](https://wix.com), [Red Hat](https://www.redhat.com), and the [Big Things community](https://www.meetup.com/Big-things-are-happening-here/),
+we ran several Presto summits across the world: 
+* [Tel Aviv, Israel, April 2019]({% post_url 2019-05-03-Presto-Conference-Israel %})
+* [San Francisco, USA, June 2019]({% post_url 2019-05-17-Presto-Summit %})
+* [Tokyo, Japan, July 2019]({% post_url 2019-07-11-report-for-presto-conference-tokyo %}) 
+* [Bangalore, India, September, 2019]({% post_url 2019-09-05-Presto-Summit-Bangalore %})
+* [New York, USA, December 2019](https://www.starburstdata.com/technical-blog/nyc-presto-summit-recap/) 
+
+All these events were a huge success and brought thousands of Presto users, contributors and other community members together to 
+share their knowledge.
+
+The project has been more active than ever. We completed 28 releases comprised of more than 2850 
+commits in over 1500 pull requests. Of course, that alone is not a good measure of progress, so 
+let’s take a closer look at everything that went in. And there is a lot to look at!
+
+<!--more-->
+
+# Improvements
+
+## Language Features
+
+* [`FETCH FIRST n ROWS [ONLY | WITH TIES]`](/docs/current/sql/select.html#limit-or-fetch-first-clauses) 
+  standard syntax. The `WITH TIES` clause is particularly useful when some of the rows have the same 
+  value for the columns being used to order the results of a query. Consider a case where you want to 
+  list top 5 students with highest score on an exam. if 6th person has the same score as the 5th, you 
+  want to know this as well, instead of getting an arbitrary and non-deterministic result:
+
+  ```sql
+  SELECT student_name, score
+  FROM student s JOIN exam_result e USING (student_id)
+  ORDER BY score
+  FETCH FIRST 5 ROWS WITH TIES
+  ```
+
+* [`OFFSET`](/docs/current/sql/select.html#offset-clause) syntax, which is especially useful in ad-hoc queries.
+* [`COMMENT ON <table>`](/docs/current/sql/comment.html) syntax to 
+  set or remove table comments, and ability to see table comments via `DESCRIBE` or the 
```

Comment:
We've shown comments with `DESCRIBE` forever. How about making this two sentences
```
* [`COMMENT ON <table>`](/docs/current/sql/comment.html) syntax to 
  set or remove table comments. Comments can be shown via `DESCRIBE`
  or the new `system.metadata.table_comments` table.